StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentLogoView.swift

```diff
@@ -65,12 +64,266 @@ private func CreateCircularLogoView(urlString: String) -> UIView {
     return imageView
 }
 
-private func CreateEllipsisView() -> UIView {
-    let imageView = UIImageView(image: Image.ellipsis.makeImage())
-    imageView.translatesAutoresizingMaskIntoConstraints = false
+private func CreateEllipsisView(
+    leftLogoUrl: String?,
+    rightLogoUrl: String?
+) -> UIView {
+    let ellipsisViewWidth: CGFloat = 32.0
+
+    let backgroundView = MergedLogoView(
+        leftImageUrl: leftLogoUrl,
+        rightImageUrl: rightLogoUrl
+    )
+    backgroundView.clipsToBounds = true
+    backgroundView.translatesAutoresizingMaskIntoConstraints = false
     NSLayoutConstraint.activate([
-        imageView.widthAnchor.constraint(equalToConstant: 16),
-        imageView.heightAnchor.constraint(equalToConstant: 4),
+        backgroundView.widthAnchor.constraint(equalToConstant: ellipsisViewWidth),
+        backgroundView.heightAnchor.constraint(equalToConstant: 6),
     ])
-    return imageView
+
+    let multipleDotView = MultipleDotView()
+    multipleDotView.frame = CGRect(
+        x: -multipleDotView.bounds.width + ellipsisViewWidth,
+        y: 0,
+        width: multipleDotView.bounds.width,
+        height: multipleDotView.bounds.height
+    )
+
+    backgroundView.mask = multipleDotView
+
+    // TODO(kgaidis): add code to repeat animation even when UIViewController is presented
```

Comment:
Note that the TODO's are expected, please see PR description where a lot of this code is WIP as part of an integration branch:
- https://github.com/stripe/stripe-ios/pull/3093#issue-2014979892